### PR TITLE
Error handling

### PR DESCRIPTION
**Pull request by wesanderson303**

### CODE REVIEW

File: plugins/pencilblue/controllers/admin/content/pages/page_form.js [hunk 1]
```diff
@@ -73,9 +73,9 @@ module.exports = function(pb) {
       self.ts.registerLocal('angular_objects', new pb.TemplateValue(self.getAngularObjects(tabs, results), false));
       self.ts.load('admin/content/pages/page_form', function(err, data) {
           var result = data;
-          self.checkForFormRefill(result, function(newResult) {
+          self.checkForFormRefill(result, function(err, newResult) {
```
Comment (evanadams28) [inline plugins/pencilblue/controllers/admin/content/pages/page_form.js]: Did we mean to accept an error param here and not handle it?

File: plugins/pencilblue/controllers/admin/content/articles/article_form.js [hunk 2]
```diff
@@ -75,9 +75,9 @@ module.exports = function(pb) {
       self.ts.load('admin/content/articles/article_form', function(err, data) {
           self.onTemplateRetrieved('' + data, function(err, data) {
               var result = '' + data;
-              self.checkForFormRefill(result, function(newResult) {
+              self.checkForFormRefill(result, function(err, newResult) {
```
Comment (evanadams28) [inline plugins/pencilblue/controllers/admin/content/articles/article_form.js]: Do we want to handle this passed error param if it exists?

File: plugins/pencilblue/controllers/admin/content/pages/page_form.js (hunk 1 above)
Comment (wesanderson303) [inline plugins/pencilblue/controllers/admin/content/pages/page_form.js]: Error logged and returned

File: plugins/pencilblue/controllers/admin/content/articles/article_form.js (hunk 2 above)
Comment (wesanderson303) [inline plugins/pencilblue/controllers/admin/content/articles/article_form.js]: Error logged and returned

**Pull request merged**